content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -248,6 +248,49 @@ User is expected to have `read` access to the resources referenced by `paramKind
 Note that if a resource in `paramKind` fails resolving via the restmapper, `read` access to all
 resources of groups is required.
 
+#### `paramRef`
+
+The `paramRef` field specifies the parameter resource used by the policy. It has the following fields:
+
+- **name**: The name of the parameter resource.
+- **namespace**: The namespace of the parameter resource.
+- **selector**: A label selector to match multiple parameter resources.
+- **parameterNotFoundAction**: (Required) Controls the behavior when the specified parameters are not found.
+
+  - **Allowed Values**:
+    - **`Allow`**: The absence of matched parameters is treated as a successful validation by the binding.
+    - **`Deny`**: The absence of matched parameters is subject to the `failurePolicy` of the policy.
+
+One of `name` or `selector` must be set, but not both.
+
+**Note:** The `parameterNotFoundAction` field in `paramRef` is **required**. It specifies the action to take when no parameters are found matching the `paramRef`. If not specified, the policy binding may be considered invalid and will be ignored or could lead to unexpected behavior.
```

Comment:
We could use `note` [shortcode](https://gohugo.io/content-management/shortcodes/) here. 
Refer the guidance on how to use it : https://kubernetes.io/docs/contribute/style/style-guide/#note